storage/fakestorage_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package storage
+
+import (
+	"context"
+	"io"
+	"os"
+	"testing"
+
+	"cloud.google.com/go/storage"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPrepareFakeServer(t *testing.T) {
+	// given
+	indexFile := "testdata/search-index-all-1.json"
```

Comment:
Could we add a comment here about how to regenerate this file? I guess this has been downloaded from an actual registry?

Reply:
Added